meta/releasing.md

```diff
@@ -4,6 +4,51 @@ The whole specification is now released as a single unit/artifact. This document
 the process for releasing the specification and a description of how the (public)
 machinery works.
 
+## Timeline
+
+The spec is released each calendar quarter. The target release dates are within the
+following ranges:
+
+* Q1: January 20-27 (critically, before FOSDEM).
+* Q2: May 20-27.
+* Q3: August 20-27.
+* Q4: November 1-15 (before recurring November conflicts).
+
+The SCT aims to have dates picked out by:
+
+* Q1: January 10.
+* Q2: May 1.
+* Q3: August 1.
+* Q4: October 15.
```

Comment:
this is all very formal... Honestly I'd have thought "The SCT will normally announce a release date 2-3 weeks before the release" would be fine.  Does anyone other than ourselves actually care exactly what release date is, to that degree?

Reply:
This doc is written for us, it just happens to be publicly accessible. We don't really have a good place to put process stuff like this, so I've stuck it in the release procedures. No one but us cares how to cut a release either :D